src/components/CustomTabBar.tsx

```
@@ -14,6 +22,81 @@ export default function CustomTabBar({
+
+  // Animated styles for active button
+  const animatedActiveButtonStyle = useAnimatedStyle(() => ({
+    transform: [
+      { translateX: activeButtonPosition.value },
+      { scale: animatedScale.value },
+    ],
+  }));
+
+  // Animated styles for tab icons
+  const getAnimatedIconStyle = (index: number) => {
+    return useAnimatedStyle(() => {
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)